app/views/content/steps-to-become-a-teacher/_step_7_apply_for_a_teaching_role.erb

```diff
@@ -0,0 +1,5 @@
+<p>Some teacher training providers recommend you start thinking about job applications quite early in your teacher training year.</p>
+
+<p>Schools start to advertise their vacancies from January.</p>
+
+<p>But don't worry – you can get tips on <a href="https://teaching-vacancies.campaign.gov.uk/application-advice/">how to apply for teaching jobs</a>.</p>
```

Comment:
@EllieNodder Saying 'don't worry' sounds like there is something to worry about? Seems a bit odd?